### PR TITLE
1022994

### PR DESCRIPTION
**Pull request by 1022994**

### CODE REVIEW

File: whoami.c [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #include <stdio.h>
 
 int main() {
+	printf("Student's ID: 1022994");
```
Comment (jrjang2) [inline whoami.c]: Leave an empty line before ```return 0;```.

**Comment by jrjang2** [comment]
There is a conflict now. Please solve this issue before doing pull request. If you have any questions, you can ask here.

Tip:
1. Fork this project (jrjang2/lab) again to obtain the last git tree.
2. Update (```git pull --rebase```) your working directory.
3. Fix conflict and commit.
4. pull request.

**Comment by 1022994** [comment]
老師 不好意思請問一下
為什麼我merge之後  東西卻越來越多呢？？

**Comment by jrjang2** [comment]
"東西"是指? 程式碼嗎? 還是commit?

**Comment by jrjang2** [comment]
Merge all commits to one before doing pull request.

**Pull request closed without merging**